code/modules/mob/living/carbon/human/human_powers.dm

```diff
@@ -1,6 +1,31 @@
 // These should all be procs, you can add them to humans/subspecies by
 // species.dm's inherent_verbs ~ Z
 
+/mob/living/carbon/human/proc/tie_hair()
+	set name = "Tie Hair"
+	set desc = "Style your hair."
+	set category = "IC"
+
+	if(h_style)
+		var/datum/sprite_accessory/hair/hair_style = hair_styles_list[h_style]
+		var/selected_string
+		if(hair_style.length <= 1)
+			to_chat(src, "<span class ='warning'>Your hair isn't long enough to tie.</span>")
+			return
+		else
+			var/list/datum/sprite_accessory/hair/valid_hairstyles = list()
+			for(var/hair_string in hair_styles_list)
+				var/list/datum/sprite_accessory/hair/test = hair_styles_list[hair_string]
+				if(test.length >= 2 && ("Human" in test.species_allowed))
```

Comment:
let's do this instead:
```suggestion
				if(test.length >= 2 && (species.name in test.species_allowed))
```

This way you can make it usable for other species by simply giving them the verb. Like Tajarans.

Reply:
That would probably require some working with adding in length vars for the Tajaran hairstyles too, which all seem to use their inherited value of 1 (they're unset).
Might have to touch in with @LadyFowls regarding hair tugging and whoever else is involved with Tajaran lore regarding hair fashion.

Reply:
I should also probably make mention that the code on Polaris and BS12 makes use of flags to track tieable hair, without a length variable. I was wondering if doing that would be redundant though when we have another system going for hair length.

Reply:
Otherwise, the changes have been committed.